nautobot_golden_config/tests/test_views.py

```diff
@@ -292,3 +295,69 @@ def test_csv_export_with_filter(self):
         device_names_in_export = [entry.split(",")[0] for entry in csv_data[1:]]
         device_names_in_site_1 = [device.name for device in devices_in_site_1]
         self.assertEqual(device_names_in_export, device_names_in_site_1)
+
+
+# pylint: disable=too-many-ancestors
+class ConfigPlanTestCase(
+    ViewTestCases.GetObjectViewTestCase,
+    ViewTestCases.GetObjectChangelogViewTestCase,
+    ViewTestCases.ListObjectsViewTestCase,
+    # ViewTestCases.CreateObjectViewTestCase,
```

Comment:
Commented out for a reason? Or can we clean these up

Reply:
It is commented out as a way to be explicit that we are deliberately not testing Create since the Create view methods are not enabled. I can add a comment that explains that or I can clean them up - let me know which you prefer.